resource/abstractions/auditing_rules.md

```diff
@@ -0,0 +1,478 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Abstractions
+---
+
+# Auditing Rules
+
+General Resource Profile Administration functions for manipulating a General Resource Profile's Auditing Rules. 
+{: .fs-6 .fw-300 }
+
+## `ResourceAdmin.get_audit_rules()`
+
+```python
+def get_audit_rules(self, resource: str, class_name: str) -> Union[dict, bytes, None]:
+```
+
+#### 📄 Description
+
+Check the **Audit Rules** for a general resource profile.
```

Comment:
```markdown
Get the **Audit Rules** for a general resource profile.
```

Reply:
Agree, change made to all occurrences of **Audit Rules** to **Auditing Rules** as well.